src/loaders/JsonImageInfoLoader.ts

```diff
@@ -192,7 +197,7 @@ class JsonImageInfoLoader extends ThreadableVolumeLoader {
     const requestedChannels = loadSpec.channels;
     if (requestedChannels) {
       // If only some channels are requested, load only images which contain at least one requested channel
-      images = images.filter(({ channels }) => channels.some((ch) => ch in requestedChannels));
+      images = images.filter(({ channels }) => channels.some((ch) => requestedChannels.includes(ch)));
```

Comment:
this is the key bugfix for getting the right channels to load

Reply:
(shaking fist at the sky) javascriiiiiipt!

Reply:
I had to look this one up, so in this case it was trying to check if `ch` was an index for `requestedChannels` instead of if `ch` was a value?

Reply:
right.  "in" roughly means "is a property key for", which for arrays just means the array indices.  This took me a while to track down, because ordinarily we request all channels but I made a change in w3cv to request only required channels - I kept asking for channels [0,1,2,5,7] and it consistently refused to load channel 7!